data/json/monsters/reptile_amphibian.json

```diff
@@ -416,6 +416,160 @@
     "zombify_into": "mon_meat_cocoon_small",
     "flags": [ "SEES", "SMELLS", "WARM", "VENOM", "SWIMS" ]
   },
+  {
+    "id": "mon_turtle",
+    "type": "MONSTER",
+    "name": { "str": "turtle" },
+    "description": "A common, mostly aquatic freshwater turtle, its pace ponderous and its specific species hard to determine.",
+    "default_faction": "small_animal",
+    "bodytype": "turtle",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "REPTILE" ],
+    "diff": 2,
+    "volume": "1 L",
+    "weight": "2 kg",
+    "hp": 18,
+    "speed": 15,
+    "material": [ "flesh" ],
+    "symbol": "t",
+    "color": "green",
+    "aggression": 0,
+    "morale": 10,
+    "aggro_character": false,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 2 } ],
+    "dodge": 0,
+    "vision_day": 15,
+    "vision_night": 7,
+    "harvest": "mammal_tiny",
+    "dissect": "dissect_lizard_sample_single",
+    "anger_triggers": [ "HURT" ],
+    "fear_triggers": [ "PLAYER_CLOSE" ],
+    "placate_triggers": [ "PLAYER_WEAK" ],
+    "reproduction": { "baby_monster": "mon_turtle_s", "baby_count": 15, "baby_timer": 420 },
+    "baby_flags": [ "SUMMER" ],
+    "//babies": "hard to determine an egg number for now with a generic freshwater turtle, as some species will lay hundreds and others only a couple.",
+    "upgrades": { "half_life": 34, "into": "mon_turtle_big" },
```

Comment:
Please don't use half-life for mutant animals. It's one of the things setting them apart from zombies.

Reply:
Most of what I used was copied over from the rattlesnake line, what would be the ideal thing to use instead?